jib-gradle-plugin/README.md

```diff
@@ -260,6 +261,7 @@ Property | Type | Default | Description
 `jib.baseImageCache` | `File` | `[user cache home]/google-cloud-tools-java/jib` | Sets the directory to use for caching base image layers. This cache can (and should) be shared between multiple images.
 `jib.applicationCache` | `File` | `[project dir]/target/jib-cache` | Sets the directory to use for caching application layers. This cache can be shared between multiple images.
 `jib.console` | `String` | *None* | If set to `plain`, Jib will print plaintext log messages rather than display a progress bar during the build.
+`jib.alwaysCacheBaseImage` | `boolean` | `false` | When building to a registry, by default Jib will skip downloading and caching base image layers that already exist in the target registry. Setting to false causes Jib to always cache layers locally.
```

Comment:
Hmm, do you think we actually need to document this? According to your design description it's a rare case that this will ever be needed, maybe we should avoid cluttering up the docs and only document it through the error message.

Reply:
Makes sense. Initially I thought this is some sort of breaking change, but now I think this is more of an internal detail that doesn't really surface to users.